source/guides/guides/stubs-spies-and-clocks.md

```diff
@@ -39,9 +39,9 @@ You can refer to each of these libraries documentation for more examples and exp
 
 ## Stubs
 
-A stub is a way to modify a function and force its behavior to be controlled by you (the programmer).
+A stub is a way to modify a function and delegate control overs its behavior to you (the programmer).
 
-A stub is most commonly used in a unit test but is still useful during some integration / e2e tests.
+A stub is most commonly used in a unit test but is still useful during some integration/e2e tests.
```

Comment:
Usually it is neater to only surround a forward slash with spaces when separating multi-word collocations.

E.g.
integration testing / code linting
but
integration/e2e tests